src/main/java/com/javachallengers/simpson/model/SimpsonCharacter.java

```diff
@@ -2,11 +2,19 @@
 
 import java.io.Serializable;
 import java.time.LocalDate;
-import java.util.Objects;
 
 import org.springframework.data.annotation.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
 
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
 @Document(collection = "simpson-character")
 public class SimpsonCharacter implements Serializable {
```

Comment:
Do we need to open getters and setters and a builder?

The ideal is to isolate our fields as much as possible. Can we change from @Data to @Getter?

Reply:
Done.